### PR TITLE
LRDOCS-8096 - Guest Checkout revisions

### PR DESCRIPTION
**Pull request by justinchoi001**

### CODE REVIEW

File: docs/commerce/2.x/en/store-administration/guest-checkout/guest-checkout-overview.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+Users can allow guests to submit orders without first having to sign in. The Guest Checkout function can be enabled or disabled for each [Channel](../managing-a-catalog/creating-and-managing-products/channels/introduction-to-channels.md). See [Enabling Guest Checkout](./enabling-guest-checkout.md) to learn more. Otherwise, by default, all users must login to create orders.
+
+Once Guest Checkout has been enabled, in order for their orders to be processed, guests must enter their email address during the checkout process. (Normally, the email address field does not appear when an authenticated user goes through the checkout process.)
+
+![Guest must enter their email address to proceed.](./guest-checkout-overview/images/01.png)
+
+After providing their email, Guests are then redirected to log in or create an account.
+
+![Guests must either log in or create an account t proceed.](./guest-checkout-overview/images/03.png)
+
```
Comment (jrhoun) [inline docs/commerce/2.x/en/store-administration/guest-checkout/guest-checkout-overview.md:12]: Hey @justinchoi001 parts of this seem wrong to me. The way this reads is that it makes it sound like an unauthenticated (guest) user can _start_ the checkout process, but then later are...

> After providing their email, Guests are then redirected to log in or create an account.

Then required to log in or create an account? That doesn't make sense, how is it guest checkout if we're requiring a log in? I reviewed the requirements ticket (https://issues.liferay.com/browse/COMMERCE-2402) and how it reads to me is:

Unauthenticated (guest) users can begin checkout, are required to provide an email address, are _prompted_ to log in / create an account OR continue as a guest. That directly seems to contradict what you're saying on line 11:

> Guests must either log in or create an account t proceed

Can you double back and confirm that this is intended, or update the docs here?

Comment (justinchoi001) [reply docs/commerce/2.x/en/store-administration/guest-checkout/guest-checkout-overview.md:12]: Hi JR, I'll check this out.

Comment (justinchoi001) [reply docs/commerce/2.x/en/store-administration/guest-checkout/guest-checkout-overview.md:12]: I've updated the [alt] text.

**Comment by jrhoun** [comment]
Merged. Thanks. See changes here: https://github.com/liferay/liferay-learn/commit/2474d76983d943b322b177a532ec130736835aa1

**Pull request closed without merging**